services/f-droid/f-droid.tester.js

```diff
@@ -31,26 +31,33 @@ const testJson = `
 const base = 'https://f-droid.org/api/v1'
 const path = `/packages/${testPkg}`
 
-t.create('Package is found')
+t.create('f-droid.org: Package is found')
   .get(`/v/${testPkg}.json`)
   .intercept(nock => nock(base).get(path).reply(200, testJson))
   .expectBadge({ label: 'f-droid', message: 'v0.2.7' })
 
-t.create('Package is found (pre-release)')
+t.create('f-droid.org: Package is found (pre-release)')
   .get(`/v/${testPkg}.json?include_prereleases`)
   .intercept(nock => nock(base).get(path).reply(200, testJson))
   .expectBadge({ label: 'f-droid', message: 'v0.2.11' })
 
-t.create('Package is not found with 403')
+t.create('f-droid.org: Package is not found with 403')
   .get(`/v/${testPkg}.json`)
   .intercept(nock => nock(base).get(path).reply(403, 'some 403 text'))
   .expectBadge({ label: 'f-droid', message: 'app not found' })
 
-t.create('Package is not found with 404')
+t.create('f-droid.org: Package is not found with 404')
   .get('/v/io.shiels.does.not.exist.json')
+  .intercept(nock =>
+    nock(base)
+      .get('/packages/io.shiels.does.not.exist')
+      .reply(404, 'some 404 text'),
+  )
```

Comment:
why did we replace the live test with a mock here? we generaly prefer live tests when we can use them

Reply:
this applies to new added tests as well that are not live

Reply:
I replaced it because all other tests are mocked, only this one was a live test for whatever reason. That seems inconsistent.

At the end of the test logic we have an additional live test to check the API

Reply:
My bad, did not notice that, you are right this is the outlier but i would prefer to keep it live so we get a pulse from upstream that its alive.

I tried to search the history but could not find why this service has most tests mocked, I would not require a change to existing tests to be live but its something we might want to look at in future PR.
My suggestion is keeping this one live and having at least one live test for a stable repo.

Reply:
There are two live test for repo testing, one for the default `f-droid.org` and one for custom repos (`apt.izzysoft.de` in this case).

So should I revert the change regarding this one test?

Reply:
Missed that we have `'f-droid.org: The real api did not change'` for f-droid.org.
so there is no need to change this specific test into a live one.
But we don't have a live test for the custom repo, notice that in the last test `'custom repo: The real api did not change'` the custom repo is not used.

Reply:
Whoops, that's an error